pyproject.toml

```diff
@@ -114,6 +114,8 @@ upload-dir = 'doc/_build/html'
 
 [tool.pydocstyle]
 match = '(?!coverage).*.py'
+convention = "numpy"
+add-ignore = ["D404"]
```

Comment:
"This" (sorry couldn't help myself) 
ignore looks weird to me.  You didn't add any docstrings, just reformatted whitespace. Did we need to add this because the numpy style changes the meaning to apply to sections as well as the entire docstring?  I see some sections have "This" as the first word.

A comment here would help us remember why we are ignoring it.

Reply:
Indeed, `D404` is raised multiple times in the code base.

However, I found no such rule in [numpydocstyle checks ](https://numpydoc.readthedocs.io/en/latest/validation.html#), so I believe this is a choice from `pydocstyle` devs. Moreover, `D404` is ignored for both `pep257` and `google` conventions, so I don't think there is a consensus on that particular error code.

If you agree, I will add a comment in such a way.

Reply:
I didn't scroll down far enough on the pydocstyle error codes docs.  I think it is fine as is!  Thanks for the further info for me to understand.